### ISSUE TITLE
Confirmation Withdraw

### ISSUE THREAD

**Issue by chambber** [blinktrade/BlinkTradeJS#39]
Hello

When I make a withdrawal request through WebSocket it asks me for the secondFactor since I have activated it in my account, and for Rest it sends me an email with a code to confirm.
There is no way you do not need any code to confirm the withdrawals, as an example in other exchanges that if they enable the API service they do not need confirmation?

**Comment by cesardeazevedo** [comment]
If i understood it correctly, you need to ask the support of your broker to remove the withdraw confirmation by email

**Comment by chambber** [comment]
Oh, right, thanks!

**Comment by LeoFCardoso** [comment]
Same issue here. Through websocket send message U6, then send message u24 with second factor from e-mail *AND* from OTP (phone token). All I can get with Foxbit (provider 4) is "Not Authorized".
I would like to test with testnet (provider 5) but my account is never "validated" nor has tBTC available for withdraw test.

**Comment by chambber** [comment]
Yes, on Foxbit I talked to them and that was their response (sorry, the conversation was in Portuguese): 
![screenshot_7](https://user-images.githubusercontent.com/11406841/40135426-b6d6b81a-591b-11e8-8341-9a944f66ad1e.png)

**Comment by cesardeazevedo** [comment]
Guys, thanks for report, i gonna take a look at it more closely and understand what's going on

**Comment by cesardeazevedo** [comment]
Ok, i've a better understanding now.

Withdrawals through the API won't work for now, so there's two possible solutions.

* Use a bitcoin address that was already used before, which won't ask for confirmation.

* Contact foxbit's support again, they probably didn't understood correctly the situation, or they didn't know the feature on the admin that they can disable withdraw confirmations, also, they don't have this procedure since this isn't actually very common, the ideal case, they would ask for you to hold a paper written "disable withdrawals confirmation".

This is their admin
<img width="278" alt="screen shot 2018-05-20 at 1 02 20 am copy" src="https://user-images.githubusercontent.com/5366959/40275596-48b6ddae-5bca-11e8-9c1f-41b09a11771e.png">

So, is absolutely possible to disable withdraw confirmations.

Those things is gonna be improved soon, let me know if you guys have any other question.
